lib/checkstl.cpp

```diff
@@ -2134,6 +2140,12 @@ void CheckStl::string_c_strConcat(const Token* tok)
     reportError(tok, Severity::performance, "stlcstrConcat", msg, CWE704, Certainty::normal);
 }
 
+void CheckStl::string_c_strStream(const Token* tok)
+{
+    std::string msg = "Passing the result of c_str() to a stream is slow and redundant.\nSolve that by directly passing the string.";
```

Comment:
If user uses `--verbose` the output will be something like:

    file1.cpp:123:8: (performance) Solve that by directly passing the string. [stlcstrStream]

please update. The first line in the message is skipped in the verbose output. We did it like this because you sometimes want to rephraze the verbose message.